src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -394,6 +407,32 @@ public static function convertWooCommerceToStorekeeperOrderStatus($wc_order_stat
         }
     }
 
+    /**
+     * @param WC_Order $databaseOrder - Order items to compare from
+     * @param $backofficeOrder - Order items to compare to
+     */
+    protected function checkOrderDiscrepancy(WC_Order $databaseOrder, $backofficeOrder): bool
+    {
+        $hasDiscrepancy = false;
+        $databaseOrderItems = $this->getOrderItems($databaseOrder);
+        $backofficeOrderItems = $backofficeOrder['order_items'];
+
+        foreach ($databaseOrderItems as $databaseOrderItem) {
+            $sku = $databaseOrderItem['sku'];
+            if (!in_array($sku, array_column($backofficeOrderItems, 'sku'), true)) {
+                $hasDiscrepancy = true;
+                break;
+            }
+        }
```

Comment:
you should at least check the `sku:quantity:ppu_wt` I can easily imaging when the order is fully different but with the same total, and your function will say no difference

Reply:
Added checking of `ppu` and `quantity` pero order items.